### ISSUE TITLE
ModuleNotFoundError: No module named 'sort'

### ISSUE THREAD

**Issue by erenozkandev** [computervisioneng/automatic-number-plate-recognition-python-yolov8#19]

**Comment by OmarElgammal1** [comment]
You will need to clone [Sort Github Repo](https://github.com/abewley/sort) and use it

**Comment by ivanciucdragos1** [comment]
> You will need to clone [Sort Github Repo](https://github.com/abewley/sort) and use it

How can I do that?

**Comment by OmarElgammal1** [comment]
You can download it from [here](https://github.com/abewley/sort/archive/refs/heads/master.zip) then extract the zip file in your directory 
@ivanciucdragos1

**Comment by keyframesfound** [comment]
@OmarElgammal1 How do I extract the zip file in my directory

**Comment by OmarElgammal1** [comment]
> @OmarElgammal1 How do I extract the zip file in my directory

use this link to download [download sort zip](https://github.com/abewley/sort/archive/refs/heads/master.zip)
then use an app to extract zip files like 7zip or WinRAR and extract it in your project directory.
@keyframesfound

**Comment by keyframesfound** [comment]
@OmarElgammal1 Yes thank you for the reply, after extracting the folder do I just put it into the directory?

**Comment by g-i-o-r-g-i-o** [comment]
> @OmarElgammal1 Yes thank you for the reply, after extracting the folder do I just put it into the directory?

yes, and rename it to sort

did you watch the video???

https://www.youtube.com/watch?v=fyJB1t0o0ms
